MediaBrowser.Common/Net/HttpRequestOptions.cs

```diff
@@ -112,7 +110,7 @@ public enum CacheMode
     }
 
     [Flags]
-    public enum CompressionMethod
+    public enum CompressionMethods
```

Comment:
Why rename this? I think enum names are generally supposed to be singular not plural

Reply:
Not if they are flag enums